pkg/plugins/golang/v3/api.go

```diff
@@ -184,10 +184,7 @@ func (p *createAPISubcommand) PostScaffold() error {
 		return err
 	}
 	if p.runMake && p.resource.HasAPI() {
-		err = util.RunCmd("Running make", "make", "generate")
-		if err != nil {
-			return err
-		}
+		fmt.Printf("Next: generate targets with:\n$ make generate\n")
```

Comment:
IHMO I would still prefer gen all always that would make the life easier for users by sure.
However, if we would like to move in this way, then we need a better explanation. 
We need to output all commands that should be executed afterwords and why. 

c/c @estroz @jmrodri

Reply:
@camilamacedo86 , I might have got the context wrong. Do you expect 'create api' to show the output message/log of 'make generate' to the user ?

Reply:
You can check the consensus reached out in the meeting: https://github.com/kubernetes-sigs/kubebuilder/issues/2188#issuecomment-852345561.

- do not gen the samples (cr) by default 
- then, add a nice msg to let the user knows that is required to gen the target to get the manifests.

So, are we not generating other files with `make generate` or will it only generate the CR? 
IHMO the most important here would output a msg with all further steps and make clear what need to be done and why after the create API. We might need to supplement the quick start as well. Could you please give a look at that and see if would be better we add a note there with this info as well? WDYT?